src/comps/rf.cc

```diff
@@ -1,10 +1,11 @@
+#include "RandLAPACK.hh"
```

Comment:
I think this can be ``#include "RandLAPACK/comps/rf.hh"``.

Reply:
done